modules/exploits/multi/script/web_delivery.rb

```diff
@@ -188,10 +188,10 @@ def gen_psh(url, *method)
       random = "#{rand_text_alphanumeric 8}.exe"
 
       # Set filename (Use random filename if empty)
-      filename = datastore['BinaryEXE-FILENAME'].blank? ? random : datastore['BinaryEXE-FILENAME']
+      filename = datastore['PSHBinary-FILENAME'].blank? ? random : datastore['PSHBinary-FILENAME']
 
       # Set path (Use %TEMP% if empty)
-      path = datastore['BinaryEXE-PATH'].blank? ? "$env:temp" : %Q('#{datastore['BinaryEXE-PATH']}')
+      path = datastore['PSHBinary-PATH'].blank? ? "$env:temp" : %Q('#{datastore['PSHBinary-PATH']}')
```

Comment:
Good catch! 👍

Reply:
I agree with you. Out of the two alternate styles you present, I would prefer the first as it only requires a single flow check. I can make this change on Monday if you wish.